openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -148,12 +148,46 @@ class Meta:
         abstract = True
 
 
+class UserPartitionGroup(models.Model):
+    """
+    Each row represents a Group in a UserPartition.
+
+    UserPartitions is a pluggable interface. Some IDs are static (with values
+    less than 100). Others are dynamic, picking a range between 100 and 2^31-1.
```

Comment:
Let's 🤞🏼 that 100 is enough... 😸

Reply:
Well, we always have negative numbers, and the 64-bit range... 😝